src/main/java/io/github/lottetreg/httpserver/Connection.java

```diff
@@ -14,35 +14,35 @@ public Connection(Socket socket) {
   public InputStream getInputStream() {
     try {
       return this.socket.getInputStream();
-    } catch(Exception e) {
-      throw new FailedToGetInputStreamException(e);
+    } catch (Exception e) {
+      throw new FailedToGetInputStream(e);
     }
   }
 
   public OutputStream getOutputStream() {
     try {
       return this.socket.getOutputStream();
-    } catch(Exception e) {
-      throw new FailedToGetOutputStreamException(e);
+    } catch (Exception e) {
+      throw new FailedToGetOutputStream(e);
     }
   }
 
   public void close() {
     try {
       this.socket.close();
-    } catch(Exception e) {
+    } catch (Exception e) {
       throw new FailedToCloseConnection(e);
     }
   }
 
-  static class FailedToGetInputStreamException extends RuntimeException {
-    FailedToGetInputStreamException(Throwable cause) {
+  static class FailedToGetInputStream extends RuntimeException {
```

Comment:
Decided to change all of my exception classes to not have "Exception" in the name